src/System.IO.FileSystem/tests/Directory/GetLogicalDrives.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Directory_GetLogicalDrives
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.AnyUnix)]
+        public void ThrowsPlatformNotSupported_Unix()
+        {
+            Assert.Throws<PlatformNotSupportedException>(() => Directory.GetLogicalDrives());
```

Comment:
Yeah, it's not a perfect match, but as it serves a similar need, is what we do in System.IO.DriveInfo, and is approximately what Mono does (e.g. https://github.com/mono/mono/blob/8892b1da779177c45811cf9158960c2ee8c263bf/mono/io-layer/io.c#L3507), we should do the same here.